src/protocols/ethereum/components/TransferReview.vue

```diff
@@ -74,41 +79,61 @@ export default defineComponent({
       });
     }
 
-    async function transfer({ amount, recipient }: { amount: string; recipient: string }) {
+    async function transfer(
+      { amount, recipient, selectedAsset }
+      : { amount: string; recipient: string; selectedAsset: IAsset },
```

Comment:
If you search for `async function transfer(` in entire project you'll find 3 similar definitions. 2 of them are using `any` for the argument. Maybe you can create one common interface for the transfer function argument?